templates/navigation/_dropdown.html

```diff
@@ -29,76 +29,58 @@
                    href="#{{ nav_section.title | slug }}-content-back"
                    data-level="2">Back</a>
               </li>
-              {% for links_section in nav_section.primary_links %}
-                {% if links_section.title %}
-                  <li class="p-navigation--list-heading">
-                    <h2 class="p-muted-heading" data-level="2">{{ links_section.title }}</h2>
-                  </li>
-                {% endif %}
-                {% for link in links_section.links %}
-                  <li>
-                    <a href="{{ link.url }}"
-                       class="p-navigation__dropdown-item"
-                       data-level="2">
-                      <strong>{{ link.title }}</strong>
-                      <br />
-                      <small class="u-text--muted">{{ link.description }}</small>
-                    </a>
-                  </li>
+              {% if nav_section.primary_links %}
+                {% for link in nav_section.primary_links.links %}
+                  {% with link = link, data_level = "2" %}
+                    {% include "navigation/partials/_list-item.html" %}
+                  {% endwith %}
                 {% endfor %}
-              {% endfor %}
+              {% endif %}
               {% if nav_section.secondary_links %}
-                {% for links_section in nav_section.secondary_links %}
-                  <li class="p-navigation--list-heading">
-                    <h2 class="p-muted-heading is-muted" data-level="2">{{ links_section.title }}</h2>
+                {% if nav_section.secondary_links.title %}
+                <li class="p-navigation--list-heading">
+                  <h2 class="p-muted-heading is-muted" data-level="2">{{ nav_section.secondary_links.title }}</h2>
+                </li>
+                {% endif %}
+                {% for link in nav_section.secondary_links.links %}
+                  <li>
+                    <a class="p-navigation--list-link" href="{{ link.url }}" data-level="2">{{ link.title }}</a>
```

Comment:
The href links are being added here for preview links, should it be removed or added to be consistent with `_preview_links.html`?

Reply:
Added, I just missed this